EchoBot.cs

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can we remove these comments. Or we can also provide reference to the open source in comments.